### PR TITLE
Added Mobile menu to my portfolio website

### PR DESCRIPTION
**Pull request by V-Blaze**
Below are the major changes in this pull request 👍 : 

- We added mobile menu active functionality.
- Added the close menu button when the menu is active.
- Added clickable navigation links to different sections of the webpage

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,11 +15,15 @@
...
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                  </div>
             <ul class="menu">
-                <li><a href="#">Portfolio</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="#">Contact</a></li>
+                <li><a href="#portfolio" class="nav-link">Portfolio</a></li>
+                <li><a href="#about" class="nav-link">About</a></li>
+                <li><a href="#contact" class="nav-link">Contact</a></li>
```
Comment (Mihndim2020) [inline index.html:26]: - [ ] Please you mobile menu is missing the following features, kindly implement them: 

  - When the user clicks (or taps) the hamburger button, the mobile menu appears.
  - When the user clicks (or taps) the close (X) button, the mobile menu disappears.
  - When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
  - When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi Team, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/Organized code
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (V-Blaze) [reply index.html:26]: Changes have been made to the index.html file to fix the mobile menu unresponsive error.
This was due to the wrong link to the main.js file

Comment (rloterh) [reply index.html:26]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Valentine,  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Nice job for using cursor pointer for the mobile menu
- Mobile menu is working as intended

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**